### ISSUE TITLE
Demo app crashes on the Android emulator

### ISSUE THREAD

**Issue by hypest** [wordpress-mobile/gutenberg-mobile#1499]
The demo app crashes on launch in the Android emunlator with:

> 2019-10-25 17:06:11.165 11196-11239/com.gutenberg A/libc: Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0 in tid 11239 (mqt_js), pid 11196 (com.gutenberg)

The app works fine on devices.

Git bisecting led me to https://github.com/wordpress-mobile/gutenberg-mobile/pull/1306 as the beginning of the issue.

**Comment by maxme** [comment]
I got that segv once. One trick to get around it:
- clear the `src/initial-html.js` file.
- open the demo app.
- restore the `src/initial-html.js`, reopen and it was working.

Before doing this, I tried to pin point the problem to a specific block or html parameter but I couldn't (it seemed to crash randomly).

**Comment by chipsnyder** [comment]
I'm getting this crash on every attempt:

```
2019-12-27 10:14:01.162 7047-7047/? A/DEBUG: pid: 6962, tid: 7029, name: mqt_js  >>> com.gutenberg <<<
2019-12-27 10:14:01.162 7047-7047/? A/DEBUG: signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0
2019-12-27 10:14:01.162 7047-7047/? A/DEBUG: Cause: null pointer dereference
```

> - clear the src/initial-html.js file.
> - open the demo app.

App is able to open

> - restore the src/initial-html.js, reopen and it was working.

App Crashes again

**Comment by maxme** [comment]
@hypest will the RN update https://github.com/wordpress-mobile/gutenberg-mobile/pull/1704 fix that one?

**Comment by hypest** [comment]
> @hypest will the RN update #1704 fix that one?

Yes, my tests so far show that the crash doesn't happen on the RN 0.61.x attempts.

**Comment by leandroalonso** [comment]
JFYI: If I delete the `wp:gallery` block from `src/initial-html.js` it does not crash the app.

**Comment by hypest** [comment]
Fixed with the upgrade to RN 0.61.5 https://github.com/wordpress-mobile/gutenberg-mobile/pull/1450
